GOVERNANCE.md

```diff
@@ -22,30 +18,226 @@ New maintainers must be nominated by an existing maintainer and must be elected
+A proposal is a design document that describes a significant change to Pinniped.
+A proposal must be sponsored (or co-authored) by at least one maintainer.
+Proposals can be submitted and reviewed by anyone in the community.
+
+## When to Submit a Proposal
+If there is significant risk with a potential feature or track of work (such as complexity, cost to implement,
+product viability, etc.), then we recommend creating a proposal for feedback and approval. If a potential
+feature is well understood and doesn't impose risk, then we recommend a standard GitHub issue to clarify the details.
+
+If you are considering creating a PR to change Pinniped's source code, and you are not sure if the change is
+significant enough to require using the proposal process, then please ask the maintainers.
+
+If you would like to simply share a problem that you are having, or share an idea for a potential feature,
+and you are not planning on designing a technical solution or submitting an implementation PR, then please feel free
+to create a standard GitHub issue instead of using the proposal process.
+
+## How to Submit a Proposal
+To create a proposal, submit a PR to this repo introducing a new subdirectory under the `proposals` directory
+with a terse name (for example, `0001_my-feature-name/`) prefixed by a monotonically incrementing proposal number.
+In that new subdirectory, create a `README.md` containing the core proposal.
+Include other files as necessary to help support understanding of the feature.
+
+To make your new proposal known to all other contributors, please send a link to your new proposal PR
+on the Kubernetes Slack in [#pinniped](https://kubernetes.slack.com/archives/C01BW364RJA)
+or via the [Pinniped mailing list](mailto:project-pinniped@googlegroups.com).
+
+Author(s) of proposals for major changes will give a time period of no less than five (5) working days
+for comment and remain cognizant of popular observed world holidays.
+
+If you don't already have a maintainer to sponsor your proposal, then reach out via Slack or the mailing list
+with a description of the problem statement. If one or more of the maintainers agrees that the problem
+statement is within the scope of the project (see [SCOPE.md](SCOPE.md)) and is appropriate to be addressed by a proposal,
+then a maintainer will be assigned as your proposal's sponsor. The sponsor can provide you with support during
+the drafting of the proposal, including sharing additional project and roadmap context as it relates to your
+problem statement, answering questions, giving feedback, etc.
+
+### Proposal Template
+The below template is an example `README.md` for a new proposal.
+Other than the high-level details at the top of the template (title, authors, status, sponsor, and approval_date)
+and the disclaimer at the top, please use whichever sections make the most sense for your proposal.
+
+```md
```

Comment:
This seems to lead to weird formatting colors.

Reply:
Thanks for catching that. It was supposed to be "markdown", not "md", for the GitHub coloring to work. Fixed.